### ISSUE TITLE
createScriptProcessor is deprecated

### ISSUE THREAD

**Issue by LuisPerisVidal** [takispig/db-meter#1]
First thanks for the code, but you code is deprecated.

https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createScriptProcessor

With your permission, the next week, I'm going to try to fix your code and send you the code.

**Comment by takispig** [comment]
You are right, feel free to make any changes and send me a PR with the updated code.  
I will try to find some time and go over it as soon as possible.
